docs/source/user_guide/charge_isotope_deconvolution.rst

```diff
@@ -58,13 +58,13 @@ second isotopic peak  is the highest in intensity and the
     :linenos:
 
     charge = 4
-    seq = AASequence.fromString("DFPIANGERDFPIANGERDFPIANGERDFPIANGER")
-    seq_formula = seq.getFormula() + EmpiricalFormula("H" + str(charge))
-    isotopes = seq_formula.getIsotopeDistribution(CoarseIsotopePatternGenerator(8))
-    print("[M+H]+ weight:", seq.getMonoWeight(Residue.ResidueType.Full, 1))
+    seq = oms.AASequence.fromString("DFPIANGERDFPIANGERDFPIANGERDFPIANGER")
+    seq_formula = seq.getFormula() + oms.EmpiricalFormula("H" + str(charge))
+    isotopes = seq_formula.getIsotopeDistribution(oms.CoarseIsotopePatternGenerator(8))
```

Comment:
@jpfeuffer do you know if we can accept this split into multiple lines? e.g., does this interfere in any way with the jupyter book generation?

Reply:
nevermind... should work. but not 100% if it helps readability :)

Reply:
I agree that these suggestions are not helping readability much. Do I need to accept every suggestion tho?

Reply:
hmm I think you can ignore those... I already added some suggestion that made sense.
BTW let's see what the CI action says. It first issued an error because of the change from an output to python block.